index.js

```diff
@@ -126,4 +129,11 @@ const randomString = function(array, username = "") {
 	return msg;
 };
 
+const setRandomActivity = function() {
+	const activities = responses.BotActivities;
+	const keys = Object.keys(activities);
+	const key = keys[Math.floor(Math.random() * keys.length)];
```

Comment:
Minor: You could make this more concise via something like:

```javascript
const entries = Object.entries(activities)
const [type, specificActivities] = entries[Math.floor(Math.random() * keys.length)];
client.user.setActivity(randomString(specificActivities), {type});
```

Reply:
Sure. Fixed!